### ISSUE TITLE
FindFilesTask is not properly stopped when running inside a Swing application.

### ISSUE THREAD

**Issue by Oliver-Loeffler** [Oliver-Loeffler/FXFileChooser#5]
Add proper shutdown mechanism to prevent FindFilesTask stopping exceptionally at Swing application shutdown.

**Comment by Oliver-Loeffler** [comment]
Something like:

``` java
public void shutdown() {
        model.getFileUpdateService().cancel();
}
```
...would help to eleminate the exceptions in case a Swing application is closed. This method could be called in a WindowListener::windowClosed method.

**Comment by Oliver-Loeffler** [comment]
`shutdown()` method implemented with 830dc36a5f6f3a3f2cefc2cb5239fbffc9319f7a.
